### ISSUE TITLE
'TM Reload' does not reload main config.yml

### ISSUE THREAD

**Issue by mibby** [Realizedd/TokenManager#18]
TokenManager v3.0.5

```/tm reload``` does not seem to reload the main config.yml. Made a change to the confirm-button configuration to remove the lore, then reloaded TokenManager in-game. The lore of the confirm button did not remove.

**Comment by mibby** [comment]
Restarted the server, it seems if the lore is missing in the ```confirm-button``` main config.yml and you have confirm-buy enabled in the shop, you can't actually shop.

```
[21:53:12] [Server thread/ERROR]: Could not pass event InventoryClickEvent to TokenManager v3.0.5-SNAPSHOT
org.bukkit.event.EventException: null
	at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:72) ~[patched_1.12.2.jar:git-Paper-1413]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:78) ~[patched_1.12.2.jar:git-Paper-1413]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[patched_1.12.2.jar:git-Paper-1413]
	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:515) ~[patched_1.12.2.jar:git-Paper-1413]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:500) ~[patched_1.12.2.jar:git-Paper-1413]
	at net.minecraft.server.v1_12_R1.PlayerConnection.a(PlayerConnection.java:1981) ~[patched_1.12.2.jar:git-Paper-1413]
	at net.minecraft.server.v1_12_R1.PacketPlayInWindowClick.a(SourceFile:33) ~[patched_1.12.2.jar:git-Paper-1413]
	at net.minecraft.server.v1_12_R1.PacketPlayInWindowClick.a(SourceFile:10) ~[patched_1.12.2.jar:git-Paper-1413]
	at net.minecraft.server.v1_12_R1.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:14) ~[patched_1.12.2.jar:git-Paper-1413]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_172]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_172]
	at net.minecraft.server.v1_12_R1.SystemUtils.a(SourceFile:46) [patched_1.12.2.jar:git-Paper-1413]
	at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:843) [patched_1.12.2.jar:git-Paper-1413]
	at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:427) [patched_1.12.2.jar:git-Paper-1413]
	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:767) [patched_1.12.2.jar:git-Paper-1413]
	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:665) [patched_1.12.2.jar:git-Paper-1413]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_172]
Caused by: java.lang.NullPointerException
	at me.realized.tokenmanager.util.inventory.ItemUtil.replace(ItemUtil.java:125) ~[TokenManager-3.0.5-SNAPSHOT.jar:?]
	at me.realized.tokenmanager.shop.ConfirmInventory.update(ConfirmInventory.java:26) ~[TokenManager-3.0.5-SNAPSHOT.jar:?]
	at me.realized.tokenmanager.shop.ShopListener.on(ShopListener.java:176) ~[TokenManager-3.0.5-SNAPSHOT.jar:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor342.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:70) ~[patched_1.12.2.jar:git-Paper-1413]
	... 16 more
```

**Comment by Realizedd** [comment]
Solved in 7fa82b0
